main.go

```diff
@@ -2,25 +2,55 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"flag"
+	
+	"github.com/golang/glog"
+	
 	"github.com/asobti/kube-monkey/config"
 	"github.com/asobti/kube-monkey/kubemonkey"
 )
 
+func glogUsage() {
+	fmt.Fprintf(os.Stderr, "usage: example -stderrthreshold=[INFO|WARN|FATAL] -log_dir=[string]\n", )
+	flag.PrintDefaults()
+	os.Exit(2)
+}
+
+func initLogging() {
+	// Check commandline options or "flags" for glog parameters
+	// to be picked up by the glog module
+	flag.Usage = glogUsage
+	flag.Parse()
+
+	if _, err := os.Stat(flag.Lookup("log_dir").Value.String()); os.IsNotExist(err) {
+		if (os.MkdirAll(flag.Lookup("log_dir").Value.String(), os.ModePerm) != nil) {
+			glog.Errorf("Failed to open custom log directory; defaulting to /tmp! Error: %v", flag.Lookup("log_dir").Value, err)
+		} else {
+			glog.Errorf("Failed to open custom log directory; attempting to create custom directory! Error: %v", flag.Lookup("log_dir").Value, err)
```

Comment:
Yeah, lets make this `Infof`, and since at this point the directory has already been created, can you change the log statement to reflect that? Something like: 

```
glog.Infof("Created custom log directory as ...")
```

Reply:
Did it! 
Line 30 is an error though. If kube-monkey doesn't have the permissions to create the directory or if it fails for some other reason, then it will be an error.